lib/active_shipping/carriers/fedex.rb

```diff
@@ -137,7 +137,7 @@ class FedEx < Carrier
     DEFAULT_LABEL_STOCK_TYPE = 'PAPER_7X4.75'
 
     # Available return formats for image data when creating labels
-    LABEL_FORMATS = ['DPL', 'EPL2', 'PDF', 'ZPLII', 'PNG'] 
+    LABEL_FORMATS = ['DPL', 'EPL2', 'PDF', 'ZPLII', 'PNG']
```

Comment:
When you touch code, always make sure it's a bit better than the way you found it.